assets/css/styles.css

```diff
@@ -391,7 +391,7 @@ button.sponsor-us:hover {
 
 .landing-picture {
   display: flex;
-  align-items: flex-end;
```

Comment:
492px to 768px left aligned text was intended. I set 492px because that is a little bit after where HackDavis crosses the center line. Centering the block within this region makes it seem like there is too much space around the text.